### DISCUSSION TITLE
How to enable automerge for some paths of monorepo?

### DISCUSSION THREAD

**Discussion by makarov-roman** [renovatebot/renovate#26943] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

 37.162.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

Hi, I have a monorepo for IaC infra.
There we define infra for multiple clusters and we want to have different automerge rules depending on the path.
atm the automerge is disabled and changes are grouped by 
```json
"commitMessagePrefix": "renovate:",
"commitMessageTopic": "{{{parentDir}}}",
"commitMessageExtra": "",
"commitMessageSuffix": ""
```
but I can't find any matchSmth packageRule which will allow to select a dependency based on it's path/parentDir. Did I miss something or it's currently not supported?

Thanks in advance.

**Comment by RahulGautamSingh** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#matchfilenames
https://docs.renovatebot.com/configuration-options/#matchpackagenames

**Reply by makarov-roman** [reply]
ah, right. thanks
